spec/lib/reporting/drop_off_report_spec.rb

```diff
@@ -144,18 +187,18 @@ def expected_tables(strings: false)
       [
         ['Step', 'Unique user count', 'Users lost', 'Dropoff from last step',
          'Users left from start'],
-        ['Welcome (page viewed)'] + string_or_num(strings, 5),
-        ['User agreement (page viewed)'] + string_or_num(strings, 5, 0, 0.0, 1.0),
-        ['Capture Document (page viewed)'] + string_or_num(strings, 4, 1, 0.2, 0.8),
-        ['Document submitted (event)'] + string_or_num(strings, 4, 0, 0.0, 0.8),
-        ['SSN (page view)'] + string_or_num(strings, 3, 1, 0.25, 0.6),
-        ['Verify Info (page view)'] + string_or_num(strings, 2, 1, 0.3333333333333333, 0.4),
-        ['Verify submit (event)'] + string_or_num(strings, 2, 0, 0.0, 0.4),
-        ['Phone finder (page view)'] + string_or_num(strings, 2, 0, 0.0, 0.4),
-        ['Encrypt account: enter password (page view)'] + string_or_num(strings, 1, 1, 0.5, 0.2),
-        ['Personal key input (page view)'] + string_or_num(strings, 1, 0, 0.0, 0.2),
-        ['Verified (event)'] + string_or_num(strings, 1, 0, 0.0, 0.2),
-        ['Workflow Complete - Total Pending'] + string_or_num(strings, 0),
+        ['Welcome (page viewed)'] + string_or_num(strings, 8),
+        ['User agreement (page viewed)'] + string_or_num(strings, 8, 0, 0.0, 1.0),
+        ['Capture Document (page viewed)'] + string_or_num(strings, 7, 1, 0.125, 0.875),
+        ['Document submitted (event)'] + string_or_num(strings, 7, 0, 0.0, 0.875),
+        ['SSN (page view)'] + string_or_num(strings, 6, 1, 0.14285714285714285, 0.75),
+        ['Verify Info (page view)'] + string_or_num(strings, 5, 1, 0.16666666666666666, 0.625),
```

Comment:
some of these might be easier written more like fractions
```suggestion
        ['SSN (page view)'] + string_or_num(strings, 6, 1, 1.0/7, 0.75),
        ['Verify Info (page view)'] + string_or_num(strings, 5, 1, 1.0/6, 0.625),
```